### DISCUSSION TITLE
Trying to enable with Bun

### DISCUSSION THREAD

**Discussion by mmarrius** [FortAwesome/Font-Awesome#20182] [Q&A] [answered]
My .npmrc contains
```
@fortawesome:registry=https://npm.fontawesome.com/
//npm.fontawesome.com/:_authToken=XX
```
which works great with `yarn`. However, I want to try it with `bun` which fails to install the package.
I've created a `.bunfig.toml` file where I added a similar config

```
[install.scopes]
"@fortawesome" = { token = "XX", url = "https://npm.fontawesome.com/" }

```
which ends up in a `error: package "@fortawesome/fontawesome-pro" not found registry.npmjs.org/@fortawesome%2ffontawesome-pro 404` when running `bun install`

Am I missing something from the configuration?

I'm running on Linux with Bun 1.1.0

**Comment by mmarrius** [comment, marked as answer]
Seems like the file name was incorrect. `bunfig.toml` worked just fine and managed to install the package

**Comment by DaSchTour** [comment]
Would be nice to have this in the official documentation
